src/js/jsx/sections/style/Type.jsx

```diff
@@ -63,36 +63,55 @@ define(function (require, exports, module) {
                 return collection.pluck(document.layers.selected, "opacity");
             };
 
+            if (!Immutable.is(this.state.postScriptMap, nextState.postScriptMap)) {
+                if (nextState.postMap) {
+                    // The list of all selectable type faces
+                    var typefaces = nextState.postScriptMap
+                        .entrySeq()
+                        .sortBy(function (entry) {
+                            return entry[0];
+                        })
+                        .map(function (entry) {
+                            var psName = entry[0],
+                                fontObj = entry[1];
+
+                            return {
+                                id: psName,
+                                title: fontObj.font
+                            };
+                        })
+                        .toList();
+
+                    this.setState({
+                        typefaces: typefaces
```

Comment:
Oh, I remember now why this isn't in the font store. It's because what we're building here is explicitly a view-only data structure: the options, in a particular format for `Datalist` and `Select`. It used to be that we recalculated this on every flux state update, and it seemed less bad to move it here. So, that's what I'm sticking with unless you want to argue about it more!